### DISCUSSION TITLE
After making Copilot buy 60 days no free time

### DISCUSSION THREAD

**Discussion by gaoxk1987** [community/community#51809] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I purchased GitHub Copilot service today and was promised a 2-month free trial during checkout. However, after being charged, I noticed that the subscription only lasts for one year and there is no free trial included.

**Comment by Akash1134** [comment]
Hi @gaoxk1987 , the trial period is not included as a part of your Copilot subscription. Once the subscription ends and if a user chooses to not renew it, the account will be converted to a trial account (with the remaining trial period) Thanks!

**Reply by gaoxk1987** [reply]
Do you mean that after my one-year subscription expires, if I don’t renew it, will I still have a 2-month trial experience? For example, my one-year subscription expires on April 3, 2024, can I still use it if I don’t renew after that date?

**Reply by Akash1134** [reply]
For account-specific details and renewal info, please contact our [Support team](https://support.github.com/request). Thanks!
